frontend/src/components/create-user-page/CreateUser.scss

```
@@ -0,0 +1,44 @@
+@import "../../constants.scss";
+
+.create-user-form-heading {
+  text-align: center;
+}
+
+.create-user-form-field {
+  min-height: 55px;
+  width: 300px;
+}
+
+.create-user-form-field__label {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.create-user-form-field__input {
+  width: 60%;
+  padding: 5px;
+}
```

You can avoid needing any classes at all by doing:
```suggestion

  label {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
  }

  input {
    width: 60%;
    padding: 5px;
  }
}
```

---

done, thanks

---

please note that in the tsx I have now removed the className properties from the input and label tags